Zend/tests/bug65579.phpt

```diff
@@ -25,5 +25,5 @@ array(2) {
   [0]=>
   string(10) "testMethod"
   [1]=>
-  string(25) "testmethodfromparenttrait"
+  string(25) "testMethodFromParentTrait"
```

Comment:
This is solving https://bugs.php.net/bug.php?id=74939, right?

Reply:
Looks like it, thanks for pointing it out.